rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2370,6 +2371,8 @@ public boolean changeStartAt(String sessionId, String jobId, String startAt) thr
         Map<String, Object> schedulerProperties;
         try {
             schedulerProperties = scheduler.getSchedulerProperties();
+            schedulerProperties.putAll(WebProperties.getPropertiesAsHashMap());
+            schedulerProperties.putAll(PortalConfiguration.getPropertiesAsHashMap());
```

Comment:
Did you check that the properties do not have the same key ?

Reply:
yes they never have the same key